src/components/Navbar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../styles.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.